Updated write-pages.md (#19843)
gatsbyjs/gatsby: docs/docs/write-pages.md

1. `static-entry.js` produces a `page-renderer.js` webpack bundle that generates the HTML for a path. It requires `data.json` and uses the `pages` to lookup the page for the page.
2. To get the `jsonName` from the page object, and uses it to construct a resource path for the actual json result by looking it up in `data.json.dataPaths[jsonName]`.

> Now that you've written out page data, you can start on the [Webpack section](/docs/webpack-and-ssr/).

Reviewer: This is in the Gatsby Internals section, and the subject is Gatsby itself rather than the reader. It needs to be updated before we can merge it, so the text is still correct!
```suggestion
Now that Gatsby has written out page data, it can start on the [Webpack section](/docs/webpack-and-ssr/).
```
